source/common/protobuf/protobuf.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "google/protobuf/descriptor.h"
+#include "google/protobuf/descriptor.pb.h"
+#include "google/protobuf/io/coded_stream.h"
+#include "google/protobuf/io/zero_copy_stream.h"
+#include "google/protobuf/io/zero_copy_stream_impl_lite.h"
+#include "google/protobuf/message.h"
+#include "google/protobuf/repeated_field.h"
+#include "google/protobuf/service.h"
+#include "google/protobuf/stubs/status.h"
+#include "google/protobuf/text_format.h"
+#include "google/protobuf/util/json_util.h"
+#include "google/protobuf/util/message_differencer.h"
+#include "google/protobuf/util/time_util.h"
+#include "google/protobuf/util/type_resolver.h"
+#include "google/protobuf/util/type_resolver_util.h"
+
+namespace Envoy {
+
+namespace Protobuf = google::protobuf;
+namespace ProtobufWkt = google::protobuf;
+
+namespace ProtobufTypes {
+
+typedef std::string String;
+
+inline const String ToString(const std::string& s) { return s; }
```

Comment:
Yes, this is my assumption, today it's not used for anything on the performance critical path either.